test/functional/keyrings/raw/test_raw_aes.py

```diff
@@ -32,7 +32,7 @@
 
 _ENCRYPTION_CONTEXT = {"encryption": "context", "values": "here"}
 _PROVIDER_ID = "Random Raw Keys"
-_KEY_ID = b"5325b043-5843-4629-869c-64794af77ada"
+_KEY_ID = "5325b043-5843-4629-869c-64794af77ada"
```

Comment:
nit: for consistency, let's call this `_KEY_NAME` (and probably rename `_PROVIDER_ID` to `_KEY_NAMESPACE`) since they're substantively different now.

ditto in the raw RSA tests